app/views/manage_consents/triage.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :before_main do %>
+  <%= render AppBacklinkComponent.new(
+        href: back_link_path,
+        name: "consent page",
+      ) %>
+<% end %>
+
+<% page_title = "Is it safe to vaccinate?" %>
+
+<%= h1 page_title: do %>
+  <span class="nhsuk-caption-l nhsuk-u-margin-top-2">
+    <%= @patient.full_name %>
+  </span>
+  <%= page_title %>
+<% end %>
+
+<%= form_for @consent, url: wizard_path do |f| %>
+  <%= f.fields_for @triage do |ff| %>
```

Comment:
Hmm, I get why we're doing this, but it's a bit weird

Reply:
I've tried to remove the weirdness in [939ff5d](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/1388/commits/939ff5dab264c3ce1865548043e887963ebdfa30), could you please take another look?